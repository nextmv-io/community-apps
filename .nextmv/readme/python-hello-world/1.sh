cat input.json | python3 main.py
