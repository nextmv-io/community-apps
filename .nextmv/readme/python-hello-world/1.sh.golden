{
  "options": {},
  "solution": {},
  "statistics": {
    "result": {
      "value": 1.23,
      "custom": {
        "message": "Hello, world"
      }
    },
    "schema": "v1"
  }
}
